Documentation/ProjectFile/prj/chemical_system/kinetic_reactants/kinetic_reactant/t_fix_amount.md

```diff
@@ -0,0 +1,3 @@
+specify if the amount of the kinetic reactant should be fixed to the initial amount over the simulation.
+Pssible values: *true* (fixed); *false* (not fixed); default is *false*.
```

Comment:
```suggestion
Possible values: *true* (fixed); *false* (not fixed); default is *false*.
```

Reply:
I take care of looking in detail the doc description (next week) :-)

Reply:
@montoyav thanks!